From 4ed08e7074209f3ba6f28c9e07163b0b93f54f0e Mon Sep 17 00:00:00 2001
Message-ID: <4ed08e7074209f3ba6f28c9e07163b0b93f54f0e.1705854493.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1705854493.git.sharlatanus@gmail.com>
References: <cover.1705854493.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 13 Jan 2024 17:03:58 +0000
Subject: [PATCH 01/19] gnu: stellarium: Update to 23.4.

* gnu/packages/astronomy.scm (stellarium): Update to 23.4.
[inputs]: Add nlopt.

Change-Id: Id205ad84ff75a687f6c1fdababcf3040c0c55abd
---
 gnu/packages/astronomy.scm | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 1e2c3f8caa..88c68db56c 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -1277,7 +1277,7 @@ (define-public stackistry
 (define-public stellarium
   (package
     (name "stellarium")
-    (version "23.3")
+    (version "23.4")
     (source
      (origin
        (method git-fetch)
@@ -1286,7 +1286,7 @@ (define-public stellarium
              (commit (string-append "v" version))))
        (file-name (git-file-name name version))
        (sha256
-        (base32 "15kl8kn5fmmppbm93w5ki7lpjn09lhyhn245v3jc335xifcwd2vd"))))
+        (base32 "11l3lql3cxj7s7q7n74qd4rhbjy374b4c0lpm0dr8s85mfrq6fmc"))))
     (build-system cmake-build-system)
     ;; TODO: Complete documentation build and split into dedicated outputs.
     (arguments
@@ -1312,6 +1312,7 @@ (define-public stellarium
            gpsd
            indi
            libnova
+           nlopt
            openssl
            qtbase-5
            qtcharts
-- 
2.41.0

